tasks/interactive-bots.md

```diff
@@ -157,7 +166,10 @@ messages to users.
   The bot now responds to `@link` and `@resource`.
 
 
-* Edit the `handle_message`:
+* Create a `links` stream.
```

Comment:
It was intentional to have the same functionality as the followup bot at first (to show how to change keywords) and only then change to private messages and not sending to the followup stream. The fix should then address the "infinite loop" aspect, not the stream the messages go to.